### ISSUE TITLE
Question about pseudo-labels

### ISSUE THREAD

**Issue by pornosaur** [AndyJZhao/GLEM#5]
Hello there,

What kind of unlabelled nodes are included in your pseudo-labels? In a transductive setting, e.g. ogbn-arxiv/products, only test/valid nodes are considered unlabelled during training. Does it mean you generate the pseudo-labels for some of these nodes (test/valid) and add them to the training set, i.e. participating in loss calculation with the pseudo-labels prediction? Or your unlabelled nodes during training are out of valid/test set?

Thanks.

**Comment by AndyJZhao** [comment]
Sorry for the late reply. Your understanding is correct. In our experiments performed, the unlabeled nodes are selected as validation and test set. Due to the large scale of the dataset, only a portion of the nodes are sampled.

**Comment by pornosaur** [comment]
Great, thank you for clarification :)
